src/xb-database-manager.c

```diff
@@ -544,8 +513,10 @@ xb_database_manager_fetch_results (XbDatabaseManager *self,
           continue;
         }
 
-      document_object = document_to_json_object (document);
-      json_array_add_object_element (results_array, document_object);
+      gchar *string;
```

Comment:
should we move up decl for consistency? Eveywhere else in this file seems to keep them at top of function C 89 style